Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Call gzip with full path when creating manpages to fix FTBFS on squeeze.

diff -Naurp btrfs-tools.orig/man/Makefile btrfs-tools/man/Makefile
--- btrfs-tools.orig/man/Makefile	2011-09-04 19:04:14.364575681 +0200
+++ btrfs-tools/man/Makefile	2011-09-04 19:23:04.886180955 +0200
@@ -1,4 +1,4 @@
-GZIP=gzip
+GZIP=/bin/gzip
 INSTALL= install
 
 prefix ?= /usr/local
